test/Data/Abstract/Path/Spec.hs

```diff
@@ -2,19 +2,22 @@ module Data.Abstract.Path.Spec(spec) where
 
 import Data.Abstract.Path
 import SpecHelpers
+import qualified System.Path as Path
 
 spec :: Spec
 spec = parallel $
   describe "joinPaths" $ do
     it "joins empty paths" $
-      joinUntypedPaths "" "" `shouldBe` "."
+      go Path.currentDir Path.currentDir $ Path.currentDir
```

Comment:
This is a bit different from the original one.  The original one is
```
Path.currentDir Path.</> Path.relPath "."
```
The current one drops the ` Path.relPath "."`.